packets.go

```diff
@@ -41,13 +41,19 @@ type EncryptionHeader struct {
 	seqno           packetSeqno
 }
 
-// encryptionBlock contains a block of encrypted data. It contains
+// encryptionBlockV1 contains a block of encrypted data. It contains
 // the ciphertext, and any necessary authentication Tags.
-type encryptionBlock struct {
+type encryptionBlockV1 struct {
 	_struct            bool                   `codec:",toarray"`
 	HashAuthenticators []payloadAuthenticator `codec:"authenticators"`
 	PayloadCiphertext  []byte                 `codec:"ctext"`
-	seqno              packetSeqno
```

Comment:
I notice this (and the other packet types) embed `seqno`, presumably for use during decryption. Seems cleaner to remove them, though?